concepts/Adrenaline_dosage.yml

```diff
@@ -0,0 +1,17 @@
+description: ''
+identifiers:
+  loinc: ''
+  snomed: ''
+lower_limit: ''
+mapper:
+- klass: MimicObservationMapper
+  params:
+    constraints:
+      itemid: '221289'
+    schema: mimiciv_icu
+    table: inputevents
+  source: mimiciv
+  unit: mg/time
```

Comment:
'time' ist keine unit die wir konvertieren können. Ich hab da grob drüber geschaut und es sieht so aus als würde das item grundsätzlich in 'mcg/kg/min'? mcg soll dabei wahrscheinlich auch ug sein?